homeassistant/components/litterrobot/strings.json

```diff
@@ -59,6 +62,17 @@
       "food_level": {
         "name": "Food level"
       },
+      "hopper_status": {
+        "name": "Hopper status",
+        "state": {
+          "enabled": "Enabled",
+          "disabled": "Disabled",
+          "motor_fault_short": "Motor fault: short",
```

Comment:
Or even better, can we remove the semicolon? Motor shorted? Or something along those lines? What does short mean in this context?

Reply:
To me "Motor fault short" doesn't make a lot of sense as a sentence. The colon I think conveys the intention of this enum. There was a motor fault and it was due to an electrical short (perhaps the device can detect when the volts/amps are not as expected). So of the types of "motor faults" one can have, this was of type "short".

Reply:
Actually I've changed my mind. To be consistent with the very clear "Motor disconnected" I've gone with your suggested "Motor shorted" and also changed `motor_ot_amps` to "Motor overtorqued". This is consistent, clear, concise and likely the most helpful information to convey.